www/__mocks__/timelineHelperMocks.ts

```diff
@@ -188,18 +185,10 @@ export const mockTransitions: Array<BEMData<TripTransition>> = [
 
 const mockFilterLocation: FilteredLocation = {
   accuracy: 0.1,
-  altitude: 100,
-  elapsedRealtimeNanos: 10000,
-  filter: 'time',
-  fmt_time: '',
-  heading: 1.0,
```

Comment:
Yes, those properties are not used by the test. If we declare like `as FilteredLocation`, we don't have to list every single property.
I thought this would be a better strategy for unit tests, where we're quite often testing with 'stub' objects, or declaring invalid inputs.